opm/parser/eclipse/EclipseState/EclipseState.hpp

```diff
@@ -220,6 +222,9 @@ namespace Opm {
         void initPlyshlogTables(DeckConstPtr deck,
                                               const std::string& keywordName,
                                               std::vector<PlyshlogTable>& tableVector);
+        void initVFPProdTables(DeckConstPtr deck,
+                                              const std::string& keywordName,
```

Comment:
Strange indentation here, I was about to ask that you check that you are using spaces (and not tabs), but then I saw that the indentation of the method above is also a little off...